content/en/docs/tasks/administer-cluster/kubelet-credential-provider.md

```diff
@@ -26,11 +26,24 @@ You may be interested in using this capability if any of the below are true:
 
 This guide demonstrates how to configure the kubelet's image credential provider plugin mechanism.
 
+## Service Account Token for Image Pulls
+{{< feature-state feature_gate_name="KubeletServiceAccountTokenForCredentialProviders" >}}
+
+Starting from Kubernetes v1.33, the kubelet can be configured to send a service account token bound to the pod for which the image pull is being performed to the credential provider plugin. This allows the plugin to exchange the token for credentials to access the image registry.
+
+To enable this feature, the `KubeletServiceAccountTokenForCredentialProviders` feature gate must be enabled on the kubelet and the `tokenAttributes` field must be set in the `CredentialProviderConfig` file for the plugin. The `tokenAttributes` field contains information about the service account token that will be passed to the plugin, including the intended audience for the token and whether the plugin requires the pod to have a service account.
```

Comment:
Please help wrap these long "lines" properly as we do for other paragraphs.
https://sembr.org/

Reply:
pushed an update, LMK what you think!